### ISSUE TITLE
Model multiplicity defaults

### ISSUE THREAD

**Issue by davaya** [spdx/spdx-3-model#393]
The defaults for multiplicity values `minCount` and `maxCount` must be specified by the model files and for the model diagram.

For example, [CreationInfo](https://github.com/spdx/spdx-3-model/blob/39c03f2f254121e8f08df20b193caf0d8342478b/model/Core/Classes/CreationInfo.md) says:
```
## Properties

- specVersion
  - type: SemVer
- comment
  - type: xsd:string
  - minCount: 0
  - maxCount: 1
- created
  - type: DateTime
- createdBy
  - type: Agent
  - minCount: 1
- createdUsing
  - type: Tool
  - minCount: 0
- profile
  - type: ProfileIdentifierType
  - minCount: 1
- dataLicense
  - type: xsd:string
```
and the diagram shows
![image](https://github.com/spdx/spdx-3-model/assets/19152940/1b8b80ee-0996-4dc4-9f1d-44415014da59)

Since `specVersion`, `created`, and `dataLicense` are singular and required, and neither are shown in either the model or the diagram, we conclude:
* **minCount** default = 1
* **maxCount** default = 1

But:
`profile`, `createdUsing`, and `createdBy` have **maxCount** unspecified in the model (default to 1), but are shown as unlimited (*) in the diagram.

[Element](https://github.com/spdx/spdx-3-model/blob/main/model/Core/Classes/Element.md) has a different problem:
```
## Properties

- spdxId
  - type: xsd:anyURI
  - minCount: 1
  - maxCount: 1
- name
  - type: xsd:string
  - maxCount: 1
- summary
  - type: xsd:string
  - maxCount: 1
- description
  - type: xsd:string
  - maxCount: 1
- comment
  - type: xsd:string
  - maxCount: 1
- creationInfo
  - type: CreationInfo
  - minCount: 0
  - maxCount: 1
- verifiedUsing
  - type: IntegrityMethod
- externalReference
  - type: ExternalReference
  - minCount: 0
- externalIdentifier
  - type: ExternalIdentifier
  - minCount: 0
- extension
  - type: Extension
  - minCount: 0

```
 - **minCount** appears to default to 0, since name, summary, description, comment, and verifiedUsing are all optional but do not have a specified minCount.  All optional properties are explicitly shown as [0..1] or [0..*] in the diagram.

What are the defaults for minCount and maxCount, and where are they documented?

**Comment by zvr** [comment]
The default values, used when these are not specified, are:

- minCount: 0
- maxCount: *

Keep in mind that all the generated formats (md, html, rdf, json-schema, etc.) have both values explicit. The defaults are only relevant to our working md files.

**Comment by davaya** [comment]
My point is that creationInfo says specVersion, created and dataLicense all use both defaults, meaning [0..*].  I don't think that's what is intended, and it isn't what the diagram shows.

**Comment by goneall** [comment]
> CreationInfo says specVersion, created and dataLicense use both defaults, meaning [0..*]. That isn't the intended result. The diagram shows "created DateTime", not "created DateTime[1..1]", and requiring [1..1] to appear everywhere it is intended would clutter the diagram.

This appears to be a bug in the `CreationInfo` class definition and not a random assignment of default - IMHO.  We should update `CreationInfo` to specify required.

**Comment by goneall** [comment]
Fixed with PR #420

**Comment by davaya** [comment]
Partially fixed.  Fixed with PR #455
